### DISCUSSION TITLE
Getting On-line status from Tasmotized device

### DISCUSSION THREAD

**Discussion by Bedzaa** [arachnetech/homebridge-mqttthing#519] [General]
Hello guys,

sot sure if this is the right place to put my question, but I tried to search a lot with no success.

with this MQTT plugin You have in homebridge UI a setup portal with:
- MQTT setting : clear, access to MQTT broker, credentials
- MQT topics : my question comes here:
     - Set On: cmnd/__device__/power
     - Get On: stat/__device__/POWER
     - Get Online: empty

Im not sure of the first two entries, but the system works well. The online status is my point: anybody here having experience getting online status from a Tasmotized device?  I mean if there is any problem with e.g.Sonoff Mini (problem with power or wifi connection), the system is working without any indication of unavailability and I can switch devices ON/OFF even without them physicaly.

I would appreciate any comment or a good hint to make my system even better. I have homebridge connected to Homekit.

Thank You so much.
Bedza

**Comment by Bedzaa** [comment]
my example of working SV_02 switch Sonoff Basic (ON / OFF possible), but this switch even does NOT exist

 {
            "type": "switch",
            "name": "SV_02",
            "url": "mqtt://192.168.***.***",
            "username": "****",
            "password": "****",
            "logMqtt": true,
            "topics": {
                "getOnline": "tele/SV_02/LWT",
                "getOn": "stat/SV_02/POWER",
                "setOn": "cmnd/SV_02/power"
            },
            "integerValue": false,
            "onValue": "ON",
            "offValue": "OFF",
            "accessory": "mqttthing"
        },

**Comment by rbswift** [comment]
this works but note the online/offline status is not updated instantly. You sometimes need to restart your home app on your phone or switch between rooms to see the status change. This is the way apple does it.

```
        {
            "accessory": "mqttthing",
            "type": "lightbulb",
            "name": "Back Yard Lights",
            "topics": {
                "setOn": "cmnd/yardlights/POWER",
                "getOn": "stat/yardlights/POWER",
                "getOnline": "tele/yardlights/LWT"
            },
            "onValue": "ON",
            "offValue": "OFF",
            "onlineValue": "Online",
            "offlineValue": "Offline",
            "startPub": {
                "cmnd/yardlights/power": "",
                "stat/yardlights/POWER": ""
            },
            "logMqtt": true
        },
```
